blocks/core/ff_module/ff_module-form-checkable-list/ff_module-form-checkable-list.js

```diff
@@ -6,11 +6,20 @@ var FormLabel = require('../ff_module-form-label/ff_module-form-label');
 
 module.exports = React.createClass({
 	displayName: 'CheckableList',
+	generateClass: function(base, props) {
+		var classNames = [];
+		props = props || {};
+		classNames.push(base);
+		if (!!props.modifier) classNames.push(base + '--' + props.modifier);
+		if (!!props.classes) classNames.push(props.classes);
+		if (!!props.className) classNames.push(props.className);
+		return classNames.join(' ');
+	},
 	render: function() {
 		return <ul className='ff_module-form-checkable-list'>
 				{this.props.items.map((item)=>{
-					return <li className={'ff_module-form-checkable-list__item' + (this.props.modifier ? ' ff_module-form-checkable-list__item--'+this.props.modifier: '')}>
-					<div className={'ff_module-form-pair' + (this.props.modifier ? ' ff_module-form-pair--'+this.props.modifier: '')}>
+					return <li key={item.key} className={this.generateClass('ff_module-form-checkable-list__item', this.props.modifier)}>
```

Comment:
You'll want to pass `this.props` in here rather than `this.props.modifier` - if you look at what the function is using you can see it uses `props`